peachjam/templates/peachjam/layouts/document_list.html

```diff
@@ -48,13 +34,97 @@ <h1 class="mt-4">{% trans 'Documents' %}</h1>
           <div>
             {% block desktop-taxonomy-list %}{% endblock %}
           </div>
-          <div class="pocketlaw-hidden" data-list-facets></div>
+          <div class="pocketlaw-hidden">
+            <form method="get" id="filter-form">
```

Comment:
I would have thought the form element would wrap the entire table (facets, search, results)?